python-checks/src/test/resources/checks/django/djangoModelStrMethodCheck.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
```

Comment:
I feel we need more tests.
We need:
* One test when we inherit from a valid model, but do not override the `__str__` method (should not raise FP)
* One test when we inherit from an invalid model (should we raise the issue twice here? On the parent and on the child? This may be noisy, but probably makes sense - other option would be to raise only on whatever directly inherits from `django.db.models.Model`, but we risk raising on abstract classes whose children are correctly defined). Whatever we do should probably be documented with tests, though.
* One test when we inherit from an unknown model (should not raise FPs)

Reply:
done